src/LtiLineitem.php

```diff
@@ -143,4 +146,16 @@ public function setEndDateTime($value)
 
         return $this;
     }
+
+    public function getGradesReleased(): ?bool
+    {
+        return $this->grades_released;
+    }
+
+    public function setGradesReleased(?bool $value): static
```

Comment:
```suggestion
    public function setGradesReleased(?bool $value): self
```

PHP 7.4 strikes again!

Reply:
Ah! I pushed an update to fix this.